adhocracy4/comments_async/serializers.py

```diff
@@ -25,8 +25,7 @@ class CommentSerializer(serializers.ModelSerializer):
     class Meta:
         model = Comment
         read_only_fields = ('modified', 'created', 'id',
-                            'user_name', 'user_pk', 'user_image',
-                            'user_image_fallback', 'ratings',
+                            'user_name', 'user_pk', 'user_image', 'ratings',
```

Comment:
Could we also remove user_pk from here, since its not needed for isOwner() anymore?

Reply:
Right! Yes, we don't use it for anything else? No, what should that be. :) So yes, please remove it!

Reply:
Ok, I will do that. :smile: